lib/compress/zstdmt_compress.c

```diff
@@ -308,7 +308,7 @@ typedef struct {
     const void* srcStart;
     size_t   prefixSize;
     size_t   srcSize;
-    size_t   readSize;
+    size_t   consumed;
```

Comment:
Should be the same type as `producedCSize`.

Reply:
I don't believe so.
`producedCSize` aggregates the results from many jobs to generate full frame stats, which can be any length (streaming).
In contrast, here, these statistic are only for current job, which necessarily fits into memory.